### ISSUE TITLE
Rename PlotPoint Story Element to Scene

### ISSUE THREAD

**Issue by terrycox** [storybuilder-org/StoryCAD#47]
**Is your feature request related to a problem? Please describe.**

In a StoryBuilder outline, the plot is comprised of a series of causally linked PlotPoint story elements. StoryBuilder defines a plotpoint as either a scene or foreshortened prose (narrative exposition.) However, in common usage, a plot point is an incident that clearly affects the story's direction: an inciting incident, a midpoint, a surprise, the climax. 

**Describe the solution you'd like**

Rename PlotPoint story elements to Scene. This encompasses the ViewModel, Model, and anywhere else they're used.d

**Describe alternatives you've considered**

N/A

**Additional context**

PlotPoint references can be found via a Find In Files search. All can be replaced with one exception: In StoryReader's RecurseStoryElement(),        
                   case "PlotPoint":
                        ParsePlotPoint(node);
should be replaced with:
                    case "PlotPoint":
                        ParseScene(node);
                    case "Scene":
                        ParseScene(node);

In other words, read story outlines which use the previous naming convention as well as the new one for legacy purposes.
